<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

    <group ns="usb_cam">
    <node pkg="image_proc" type="image_proc" name="image_proc"/>
  </group>

  <node name="aruco_detection" pkg="aruco_ros_opencv" type="aruco_detection.py" />

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/aruco/marked_image_out"/>
    <param name="autosize" value="true" />
  </node>
</launch>
